src/lib/modules/Literature/LiteratureRelations.js

```diff
@@ -24,50 +24,53 @@ export default class LiteratureRelations extends Component {
   renderMultiparts = () => {
     const relations = _get(this.relations, 'multipart_monograph', []);
     if (!relations.length) return null;
+    const prefix = 'This is part of the monograph:';
```

Comment:
I guess this can be exposed as a configurable property of the component. Just to keep things consistent with the rest of the module.

Reply:
what do you mean by that?

Reply:
The component LiteratureRelations is on the 'top' components and i don't see any other top component module accepting other than props than just 'data'. So i think keeping it consistent would be keeping it like this?

Reply:
maybe it would make sense to refactor it into a unified functional component ? `renderMultiparts` and `renderSerials` are basically the same